mlir/test/Dialect/Vector/vector-transfer-flatten.mlir

```diff
@@ -131,10 +131,42 @@ func.func @transfer_read_dims_mismatch_non_contiguous_non_zero_indices(
 
 // -----
 
-// The input memref has a dynamic trailing shape and hence is not flattened.
-// TODO: This case could be supported via memref.dim
+/// The leading dynamic shapes don't affect whether this example is flattenable
+/// or not. Indeed, those dynamic shapes are not candidates for flattening anyway.
```

Comment:
Hi, not sure if on purpose to highlight comments in opposition to checks with 3 slashes `///` (which I think makes sense). Yet, other comments such as l.108 or l.167 do not implement this.

Reply:
Sorry about that, it wasn't intentional.

For context, some people use different format for comments in tests (e.g. `///` instead of `//`) as apparently their editors handle that better (i.e. use different highlighting for comment and `RUN`/`CHECK` lines).

In any case, we should always prioritise consistency - I've just updated these from `///` to `//`.